src/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <title>🎱 행운의 로또</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div id="app">
+      <div class="navbar">
+        <h1>🎱 행운의 로또</h1>
+      </div>
+      <div class="home-wrapper center">
+        <div class="home column">
+          <p class="home-title">🎱 내 번호 당첨 확인 🎱</p>
+          <div class="column">
+            <p class="sub-title">구입할 금액을 입력해주세요.</p>
+            <div class="money-input-container space-between">
+              <input id="moneyInput" class="money-input" autofocus />
+              <button id="purchaseButton" class="button">구입</button>
+            </div>
+          </div>
+
+          <div id="lottoContent" class="hidden">
+            <p class="sub-title">
+              총 <span id="count"></span>개를 구매하였습니다.
+            </p>
+            <div id="lottoList" class="lotto-list-container column"></div>
+            <div class="column">
+              <p class="sub-title">
+                지난 주 당첨번호 6개와 보너스 번호 1개를 입력해주세요.
+              </p>
+              <div class="space-between normal-font">
+                <p>당첨 번호</p>
+                <p>보너스 번호</p>
+              </div>
+              <div class="number-input-container space-between">
+                <div id="winningNumbersGroup"></div>
+                <input id="bonusNumberInput" class="number-input" />
+              </div>
+              <button id="confirmButton" class="button">결과 확인하기</button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="copy-right-wrapper center">Copyright 2023. woowacourse</div>
+
+      <div id="result" class="hidden dark-home">
+        <div class="result-modal center">
+          <button id="closeButton" class="closeButton">X</button>
+          <div class="result-title">🏆 당첨 통계 🏆</div>
+          <div class="result-list space-between">
+            <p class="result-part">일치 갯수</p>
+            <p class="resul-part">당첨금</p>
+            <p class="result-part">당첨 갯수</p>
+          </div>
+          <div id="resultListContainer"></div>
```

Comment:
```suggestion
          <div id="resultListContainer" />
```
내용이 없으면 단일태그를 사용해주세요 👍